container/images/nodejs/Dockerfile

```diff
@@ -0,0 +1,60 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# gpg keys listed at https://github.com/nodejs/node#release-team
+RUN set -ex \
+  && for key in \
+    94AE36675C464D64BAFA68DD7434390BDBE9B9C5 \
+    FD3A5288F042B6850C66B31F09FE44734EB7990E \
+    71DCFD284A79C3B38668286BC97EC7A07EDE3FC1 \
+    DD8F2338BAE7501E3DD5AC78C273792F7D83545D \
+    C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    B9AE9905FFD7803F25714661B63B535A4C206CA9 \
+    56730D5401028683275BD23C23EFEFE93C4CFFFE \
+    77984A986EBC2AA786BC0F66B01FBB92821C587A \
+  ; do \
+    gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
+    gpg --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
+    gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
+  done
+
+ENV NODE_VERSION 9.11.1
+
+RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
+  && case "${dpkgArch##*-}" in \
+    amd64) ARCH='x64';; \
+    ppc64el) ARCH='ppc64le';; \
+    s390x) ARCH='s390x';; \
+    arm64) ARCH='arm64';; \
+    armhf) ARCH='armv7l';; \
+    i386) ARCH='x86';; \
+    *) echo "unsupported architecture"; exit 1 ;; \
+  esac \
+  && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
```

Comment:
Installing Node seems so complicated. Is there a node version available in `apt`?

Reply:
Same response as in my previous PR response, I can use apt-get although we need certain versions and what if we can;t get them via apt-get, how dependent are we on the node/npm versions?

Reply:
With Python, we can use any version of Python 2.7+.  For Python 3 we need to make some changes to the runner to make it compatible.

Reply:
You can use any nodejs version provided that it is supported by Codeship: https://documentation.codeship.com/basic/languages-frameworks/nodejs/

Reply:
Just had a lookout for node or npm or nvm via `apt-get` no success

Reply:
```
root@950dc6d77b96:/srv# nvm --version
0.33.11
root@950dc6d77b96:/srv# npm --version
3.6.0
root@950dc6d77b96:/srv# node --version
v5.6.0
```